.github/workflows/main.yml

```diff
@@ -159,8 +168,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["3.2", "head"]
-        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        ruby: ["3.0", "3.1", "3.2", "head"]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
This 3x3 matrix is a lot of jobs, if it proves annoying I think it'd be fine to test only one of the versions on all platforms, and test all versions on ubuntu.